src/pages/courses.js

```diff
@@ -53,7 +285,21 @@ const PriceTag = styled.span`
   }
 `;
 
-export default function AboutPage({ data, path }) {
+const Course = ({ course: { image, title, price, link, description } }) => (
+  <div className="course">
+    <img src={image.src} alt={image.alt} />
+    <H as="h3">{title}</H>
+    <PriceTag>{price}</PriceTag>
+    <a href={link.url} target="_blank" rel="noopener noreferrer">
```

Comment:
This is the only functional change. I think it's better to open a new tab when linking to an external site, just so the user can easily come back without losing their place.